packages/autorest.python/samples/specification/multiapi/generated/azure/multiapi/sample/v1/aio/operations/_multiapi_service_client_operations.py

```diff
@@ -230,14 +235,15 @@ async def begin_test_lro(
                 product=product,
                 content_type=content_type,
                 cls=lambda x, y, z: x,
+                stream=True,
                 headers=_headers,
                 params=_params,
                 **kwargs
             )
         kwargs.pop("error_map", None)
 
         def get_long_running_output(pipeline_response):
-            deserialized = self._deserialize("Product", pipeline_response)
+            deserialized = self._deserialize("Product", pipeline_response.http_response)
```

Comment:
see 1 and 2 of https://github.com/Azure/autorest.python/pull/2589#issue-2296719842.
-  I notice that deserialization works with input `pipeline_response.http_response` for sync and async, stream and non-stream response.
- for stream pipeline_response, I find that _deserialize [works](https://github.com/Azure/autorest.python/blob/1857677a4ddc47bf5f55b49e5546dbca027ed146/packages/autorest.python/samples/specification/multiapi/generated/azure/multiapi/sample/_serialization.py#L1566-L1567) with param `pipeine_response.http_response` both for sync and async API

Reply:
i think this is a good compromise. I'd just make sure that we're only changing deserialization logic in the `begin_` lro methods that call an initial operation

Reply:
Sure.